app/scripts/modules/core/src/presentation/main.less

```diff
@@ -159,6 +159,18 @@ html {
       overflow-y: auto;
     }
   }
+  .filters-toggle {
```

Comment:
this PR adds a lot more css to `main.less`.  I'd like to move as much as this into component specific files.  This isn't a blocker for merge, it can be done at any time in the future.

Reply:
I will just do it before merge, as it is easy enough to pull to a new file.